src/dsp_tools/cli/call_action.py

```diff
@@ -194,6 +199,15 @@ def _call_ingest_xmlupload(args: argparse.Namespace) -> bool:
     )
 
 
+def _call_xml_validate(
+    # args: argparse.Namespace
+) -> bool:
+    success = xml_validate(
+        # xml_file=Path(args.xml_file), creds=_get_creds(args), imgdir=Path(args.imgdir)
+    )
```

Comment:
suggestion: for a simple (and of course not useable) setup for comparison, why not just hard-code the xml file, so that you never have to provide any params in the CLI, simply `dsp-tools xml-validate`?

Reply:
Sure I can do that. I wanted to simulate the real world implementation, but I see the point. Would you suggest I create this without any arguments in the parser?

Reply:
yes, keep it as simple as possible.